src/content/blog/dev-portal.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "The Astro Developer Portal"
+description: "Today, we’re launching the Astro Developer Portal. A platform, where theme authors and creators, can fully create, own and update their themes built for Astro."
+publishDate: "March 13, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+socialImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+import BlogSummary from "/src/components/BlogSummary.astro"
+import dashboard from "/src/content/blog/_images/dev-portal/dashboard.webp"
+
+<BlogSummary>
+Today, we're launching [the Astro Developer Portal](https://portal.astro.build). A platform, where theme authors and creators, can fully create, own and update their themes built for Astro.
+</BlogSummary>
+
+At Astro, we're dedicated to crafting solutions that not only benefit our users but also address our internal challenges. One such solution is the new Developer Portal, the new and future home for all Astro developers. Our aim with the Developer Portal is to provide a centralised platform where theme creators, and eventually integration creators, can have full control over their creations. Another goal was to test drive Astro Studio, its capabilities, and identify bugs early.
+
+Today, we're taking the new Astro Developer Portal in use. A platform, where theme authors and creators, can fully create, own and update their themes built for Astro.
+
+Check it out at [portal.astro.build](https://portal.astro.build).
+
+<BlogContentImage src={dashboard} alt="" />
+
+## Problems in our current setup
+
+The old method of adding themes to the Astro catalog was a very time-consuming and inefficient process. Theme creators were required to fill out a form, which triggered a manual process for the Astro core team. Images and content had to be extracted from Discord messages, formatted, and integrated into our theme content collection catalog. Updating themes was equally cumbersome, requiring manual repository forks and PR submissions by theme authors.
+
+You probably get that it's not the most ideal way of handling that, right?
+
+## Solution
+
+Out of identifying those problems and challenges, we came up with a new idea. A digital portal where theme creators can upload, update, and manage their themes. A core member only has to step in once, to approve a new theme, but that also can be just a click of a button. Using Astro Studio and Astro DB, we are able to handle the entire process using a web application, instead of requiring manual steps, such as creating PRs and managing content.
```

Comment:
```suggestion
Out of identifying those problems and challenges, we came up with a new idea. A digital portal where theme creators can upload, update, and manage their themes. A core member only has to step in once, to approve a new theme, but that is now a simple click of a button. Using Astro Studio and Astro DB, we are able to handle the entire process using a web application, instead of requiring manual steps, such as creating PRs and managing content.
```